modules/nf-core/mmseqs/createindex/main.nf

```diff
@@ -35,7 +35,7 @@ process MMSEQS_CREATEINDEX {
 
     stub:
     """
-    touch ${db_path_name}.idx
+    touch "${db}/\${DB_NAME}".idx
```

Comment:
`$DB_NAME` doesn't exist in this scope but the same `db_path_name` could be applied.